### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by git-taufiq**
removed redundant palu exercise about meshing

### CODE REVIEW

File: Documentation/tpv34.rst [hunk 1]
```diff
@@ -0,0 +1,168 @@
+SCEC TPV34
+==========
+
+The TPV34 benchmark uses a right-lateral, planar, vertical, strike-slip fault
```
Comment (Thomas-Ulrich) [inline Documentation/tpv34.rst:4]: features instead of uses?

File: Documentation/tpv34.rst [hunk 2]
```diff
@@ -0,0 +1,168 @@
+SCEC TPV34
+==========
+
+The TPV34 benchmark uses a right-lateral, planar, vertical, strike-slip fault 
+set in a half-space. The velocity structure is the actual 3D velocity structure
```
Comment (Thomas-Ulrich) [inline Documentation/tpv34.rst:5]: embedded instead of set?

File: Documentation/tpv34.rst [hunk 3]
```diff
@@ -0,0 +1,168 @@
+SCEC TPV34
+==========
+
+The TPV34 benchmark uses a right-lateral, planar, vertical, strike-slip fault 
+set in a half-space. The velocity structure is the actual 3D velocity structure 
+surrounding the Imperial Fault, as given by the SCEC Community Velocity Model
+`CVM-H <https://strike.scec.org/scecpedia/CVM-H>`_.
+
+.. figure:: LatexFigures/tpv34_imperialfault.png
+   :alt: The Imerial Fault.
```
Comment (Thomas-Ulrich) [inline Documentation/tpv34.rst:10]: typo

File: Documentation/tpv34.rst [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+15000 m down-dip.
+
+There is a circular nucleation zone on the fault surface. The hypocenter is 
+located 15 km from the left edge of the fault, at a depth of 7.5 km.
+
+.. figure:: LatexFigures/tpv34.png
+   :alt: Diagram of TPV34.
+   :width: 15.00000cm
+
+   Diagram of TPV34.
```
Comment (Thomas-Ulrich) [inline Documentation/tpv34.rst:30]: add reference.
change to `TPV34 overview`?

File: Documentation/tpv34.rst [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+
+The geometry is generated with `Gmsh <https://gmsh.info/>`_. All the files 
+that are needed for the simulation are provided at 
+https://github.com/SeisSol/Examples/tree/master/tpv34.
+
+.. figure:: LatexFigures/tpv34_gmshhalf.png
+   :alt: Diagram of TPV34 vertical fault in Gmsh.
+   :width: 15.00000cm
+
+   Diagram of TPV34 vertical fault in Gmsh.
```
Comment (Thomas-Ulrich) [inline Documentation/tpv34.rst:40]: changed by:
>modeled TPV34 fault in Gmsh

File: Documentation/tpv34.rst [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+
+Material
+~~~~~~~~
+
+To obtain the velocity structure for TPV34, we need to install and run 
+the SCEC Community Velocity Model `CVM-H <https://strike.scec.org/scecpedia/CVM-H>`_.
```
Comment (Thomas-Ulrich) [inline Documentation/tpv34.rst:46]: the SCEC Community Velocity Model software distribution

File: Documentation/tpv34.rst [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+
+   Diagram of TPV34 vertical fault in Gmsh.
+
+Material
+~~~~~~~~
+
+To obtain the velocity structure for TPV34, we need to install and run 
+the SCEC Community Velocity Model `CVM-H <https://strike.scec.org/scecpedia/CVM-H>`_. 
+For TPV34, we are using CVM-H version 15.1.0 and we use 
+`ASAGI <https://github.com/TUM-I5/ASAGI>`_ to map related material properties
```
Comment (Thomas-Ulrich) [inline Documentation/tpv34.rst:48]: to map the material properties variations onto the mesh.

File: Documentation/tpv34.rst [hunk 8]
```diff
@@ -0,0 +1,168 @@
...
+Material
+~~~~~~~~
+
+To obtain the velocity structure for TPV34, we need to install and run 
+the SCEC Community Velocity Model `CVM-H <https://strike.scec.org/scecpedia/CVM-H>`_. 
+For TPV34, we are using CVM-H version 15.1.0 and we use 
+`ASAGI <https://github.com/TUM-I5/ASAGI>`_ to map related material properties 
+for a SeisSol simulation. Detailed explanations are provided at 
+https://github.com/SeisSol/Examples/blob/master/tpv34/generate_ASAGI_file.sh
+
```
Comment (Thomas-Ulrich) [inline Documentation/tpv34.rst:51]: Add something like:
We generate 2 netcdf files of different spatial resolution to map more finely the 3D material properties close to the fault.
The domain of validity of each netcdf files read by ASAGI is defined by the yaml `tpv34-material.yaml` file:

File: Documentation/tpv34.rst [hunk 9]
```diff
@@ -0,0 +1,168 @@
...
+| mu\_s       | static friction coefficient    | 0.58       |        |
++-------------+--------------------------------+------------+--------+
+| mu\_d       | dynamic friction coefficient   | 0.45       |        |
++-------------+--------------------------------+------------+--------+
+| d\_c        | critical distance              | 0.18       | m      |
++-------------+--------------------------------+------------+--------+
+
+The cohesion is 1.02 MPa at the earth’s surface. It is 0 MPa at depths 
+greater than 2400 m, and is linearly tapered in the uppermost 2400 m.
+
```
Comment (Thomas-Ulrich) [inline Documentation/tpv34.rst:91]: Add:
The spatial dependence of the cohesion is straightforwardly mapped using `Easi <https://github.com/SeisSol/easi>`_.

File: Documentation/tpv34.rst [hunk 10]
```diff
@@ -0,0 +1,168 @@
...
+  [cohesion]: !FunctionMap
+    map:
+      cohesion: |
+        return -425.0*max(z+2400.0,0.0);
+
+Initial stress
+~~~~~~~~~~~~~~
+
+The initial shear stress on the fault is pure right-lateral. 
+The initial shear stress is :math:`\tau_0` = (30 MPa)(mux)
```
Comment (Thomas-Ulrich) [inline Documentation/tpv34.rst:108]: This equation does not look good.
Maybe something like:
:math:`\tau_0 = 30mux` MPa ?

File: Documentation/tpv34.rst [hunk 11]
```diff
@@ -0,0 +1,168 @@
...
+    map:
+      cohesion: |
+        return -425.0*max(z+2400.0,0.0);
+
+Initial stress
+~~~~~~~~~~~~~~
+
+The initial shear stress on the fault is pure right-lateral. 
+The initial shear stress is :math:`\tau_0` = (30 MPa)(mux) 
+The initial normal stress on the fault is :math:`\sigma_0` = (60 MPa)(mux).
```
Comment (Thomas-Ulrich) [inline Documentation/tpv34.rst:109]: same

File: Documentation/tpv34.rst [hunk 12]
```diff
@@ -0,0 +1,168 @@
...
+With mux = :math:`\mu / \mu_0`, where 
+:math:`\mu` is shear modulus and
+:math:`\mu_0` = 32.03812032 GPa.
+
+.. code-block:: YAML
```
Comment (Thomas-Ulrich) [inline Documentation/tpv34.rst:114]: In this PR. let's leave it like that, but why not using a 2d Netcdf file with the mu variation, and then generate Ts, Td and Pn based on that (see e.g. Norcia files for a model)

File: Documentation/tpv34.rst [hunk 13]
```diff
@@ -0,0 +1,168 @@
...
+        s_yz:     return 0.0;
+        s_xz:     return 0.0;
+
+
+Results
+~~~~~~~
+All examples here can be visualized in Paraview. The *output* folder contains a series of files for
+fault dynamic rupture (hdf5 and .xdmf), wavefield (hdf5 and .xdmf), on-fault receiver (.dat) and
+off-fault receivers (.dat). The fault dynamic rupture and wavefield files can be loaded in Paraview 
+directly. For example, open Paraview and then go through File > import > 'prefix'-fault.xdmf.
```
Comment (Thomas-Ulrich) [inline Documentation/tpv34.rst:162]: remove directly

File: Documentation/tpv34.rst [hunk 14]
```diff
@@ -0,0 +1,168 @@
...
+
+.. figure:: LatexFigures/tpv34_SRs_3s.png
+   :alt: Fault slip rate along-strike direction
+   :width: 15.00000cm
+
+   Fault slip rate in the along-strike direction (SRs) at 3 seconds in TPV34, illustrated in Paraview.
```
Comment (Thomas-Ulrich) [inline Documentation/tpv34.rst:168]: at a rupture time of 3 seconds

Comment (Thomas-Ulrich) [reply Documentation/tpv34.rst:168]: vizualized using ParaView.

**Review by Thomas-Ulrich — CHANGES_REQUESTED** [review]
Thx for this PR, I proposed a few changes, but overall, it is already in good shape.

File: Documentation/tpv34.rst (hunk 12 above)
Comment (git-taufiq) [reply Documentation/tpv34.rst:114]: I think this is fine since this setup made following the SCEC description.

**Review by Thomas-Ulrich — APPROVED** [review]

**Comment by Thomas-Ulrich** [comment]
@git-taufiq please merge using `squash and merge` (as you introduced and removed some large figure files that would make the repository larger to download).

**Pull request merged**